projects/packages/videopress/src/client/block-editor/hooks/use-preview/index.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * WordPress dependencies
+ */
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+/*
+ * Types
+ */
+import type { VideoPreview } from '../../../blocks/video/types';
```

Comment:
This path seems to be wrong. Can you check it? I leave a commit suggestion just in case:

```suggestion
import type { VideoPreview } from '../../../block-editor/blocks/video/types';
```